assets/stackblitz/html/index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite App</title>
+    <link rel="stylesheet" href="main.css" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <script type="module" src="/app.js"></script>
+    <script type="module" src="/main.js"></script>
+
+    <mgt-mock-provider></mgt-mock-provider>
```

Comment:
Open question: should we prefer this declarative approach for setting up the provider, or the imperative approach of having the provider set up in a script?

Reply:
That's a very good question and I prefer the second approach. Now we even have an app.js that does component registration. We could update with the mock provider!